gqlauth/settings_type.py

```diff
@@ -123,6 +124,7 @@ def override(cls, value: T) -> "DjangoSetting":
         return DjangoSetting(setting="", value=value)
 
 
+id_field = StrawberryField(python_name="id", default=None, type_annotation=StrawberryAnnotation(ID))
```

Comment:
What's the affect of this :thinking:

Reply:
While I was reviewing it, I realized that I wanted to use the id value as JWT_PAYLOAD_PK, but there was no field defined to allow me to do this. I could do it with custom, but I thought this might be needed by others as well, so I defined settings_type.id_field. Currently, the default value of the JWT_PAYLOAD_PK field is username. However, there are cases where username is not used as a unique id, and since username can be long, we think id is a more appropriate default JWT_PAYLOAD_PK. Therefore, I added id_field (JWT_PAYLOAD_PK=id_field) to make it easier for users to replace JWT_PAYLOAD_PK with id.

Reply:
ah gotcha... I don't think it's a big deal to write this yourself... 
Anyways if it is not documented no one will use it :upside_down_face:

Reply:
I just added a comment about it in the docs/settings.md file.